src/main/java/com/opencart/datamodel/WrongLoginModel.java

```diff
@@ -0,0 +1,14 @@
+package com.opencart.datamodel;
+
+public class WrongLoginModel {
+    private String wrongEmail = "uiu1@k.o";
+    private String wrongPassword = "12345678";
+
+    public String getWrongEmail() {
+        return wrongEmail;
+    }
+
+    public String getWrongPassword() {
+        return wrongPassword;
+    }
+}
```

Comment:
I think it is better to create a method, which will return a LoginModel with wrong parameters into LoginRepository.